specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefitrecommendations.json

```diff
@@ -0,0 +1,486 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CostManagementClient",
+    "description": "CostManagement management client provides access to CostManagement resources for Azure Enterprise Subscriptions.",
+    "version": "2022-10-01"
+  },
+  "tags": [
+    {
+      "name": "Benefits API",
+      "description": "API for getting benefit recommendations."
+    }
+  ],
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{billingScope}/providers/Microsoft.CostManagement/benefitRecommendations": {
+      "get": {
+        "tags": [
+          "BenefitRecommendations"
+        ],
+        "description": "List of recommendations for purchasing benefit (Reservation or Savings Plan).",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/CostManagement/"
+        },
+        "x-ms-examples": {
+          "BenefitRecommendationsBillingAccountList": {
+            "$ref": "./examples/BenefitRecommendationsByBillingAccount.json"
+          }
+        },
+        "operationId": "BenefitRecommendations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/benefitRecommendationsFilterParameter"
+          },
+          {
+            "name": "$orderby",
+            "in": "query",
+            "description": "May be used to order the recommendations by: properties/armSkuName. This implicitly filters the results by the corresponding benefit type.",
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "description": "May be used to expand the properties by: properties/usage, properties/allRecommendationDetails",
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          },
+          {
+            "$ref": "common-types.json#/parameters/apiVersionParameter"
```

Comment:
>apiVersionParameter [](http://example.com/codeflow?start=51&length=19)

If this is an ARM API as am imagining, I think you would just want to refer driectly to common-types v3 types.json ApiVersionParameter instead of having your own apiVersion parameter definition?

Reply:
Done.